src/client/datascience/raw-kernel/rawJupyterSession.ts

```diff
@@ -212,7 +198,7 @@ export class RawJupyterSession extends BaseJupyterSession {
             // Need to have connected before restarting and can't use a LiveKernelModel
             throw new Error(localize.DataScience.sessionDisposed());
         }
-        const startPromise = this.startRawSession(kernelConnection, cancelToken);
+        const startPromise = this.startRawSession(kernelConnection, -1, cancelToken);
```

Comment:
Can we use `undefined`, this way its more clear. Using -1 or 0 or the like feels odd.
E.g. in .net i know we use TimeSpan.infinity of the like for such things.

Reply:
That code isn't there. You're commenting on the older version of the file.